hazelcast-integration/spring-data-hazelcast-chemistry-sample/pom.xml

```diff
@@ -22,7 +22,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <java.version>1.8</java.version>
-        <spring-data-hazelcast.version>2.1</spring-data-hazelcast.version>
+        <!-- adding specific Hazelcast version, since spring-data-hazelcast do not support Hazelcast 4.1-SNAPSHOT version -->     
+        <hazelcast.version>3.11.7</hazelcast.version>
```

Comment:
is there a reason not to use 3.12.7?

Reply:
The major version I could find [here](https://github.com/hazelcast/hazelcast/releases) is 3.11.7, another version is [3.12.6-adobe](v3.12.6-adobe )

Reply:
You can see that 3.12.7 is latest one from download link for 3.12.z release line.
https://hazelcast.org/imdg/download/

@Holmistr @hasancelik Do you know why 3.12.7 is not listed as release but a tag on releases page?
https://github.com/hazelcast/hazelcast/releases

Reply:
verified against `3.12.7` version too. works fine.

Reply:
@mesutcelik No idea :-/